kolibri/core/content/public_api.py

```diff
@@ -50,6 +52,13 @@ def retrieve(self, request, pk=None):
         :param pk: id parent node
         :return: an object with keys for each content metadata table and a schema_version key
         """
+        try:
```

Comment:
Okey! So looking into this when the PK will be None the try block will get triggered and indeed return 400. So I think we can just update the tests to return 400. Apologies for overlooking this.